README.md

```diff
@@ -235,6 +235,13 @@ val Alpha.Companion.inOrderSealedEnum: AlphaInOrderSealedEnum
 fun Alpha.Companion.inOrderValueOf(name: String): Alpha = AlphaInOrderSealedEnum.valueOf(name)
 ```
 
+The traversal order is guaranteed to be in declaration order when the source code is nested within the sealed class.
+Sealed subclasses, however, can also be defined elsewhere in the file and package.
+
+In the general case, the default order of sealed subclasses is:
+- In source declaration order when declared as inner classes within the sealed class.
+- In qualified-name alphabetical order when not declared as an inner class within the sealed class.  
```

Comment:
This is the rule I'm proposing to have a stable order that meets the enum replacement requirement as well as makes some logic sense when there are sealed subclasses declared outside of the sealed class.

Reply:
Makes sense!  If declarations are spread out over multiple files there's no ordinality for those ones.